.docs/index.md

```diff
@@ -96,20 +78,11 @@ registered as either a SystemD or SystemV service depending upon the OS.
 $ curl -sSL https://dl.bintray.com/emccode/rexray/install | sh
 ```
 
-Refer to the User Guide's
-[installation topic](./user-guide/installation/#installation) for instructions
-on building REX-Ray from source, installing specific versions, and more.
-
 
 ### Configuring REX-Ray
 REX-Ray requires a configuration file for storing details used to communicate
 with storage providers. This can include authentication credentials and
-driver-specific configuration options. Refer to the libStorage Storage Providers
-[documentation](http://libstorage.readthedocs.io/en/stable/user-guide/storage-providers/)
-for sample configurations of all supported storage platforms. Additionally, look
-at [core properties](./user-guide/config.md#configuration-properties) &
-[logging](./user-guide/config.md#logging-configuration) for advanced
-configurations.
+driver-specific configuration options. Use the [REX-Ray Configuration Generator](http://rexrayconfig.codedellemc.com/) or refer to the libStorage Storage Providers [documentation](http://libstorage.readthedocs.io/en/stable/user-guide/storage-providers/) for sample configurations of all supported storage platforms. Additionally, look at [core properties](./user-guide/config.md#configuration-properties) & [logging](./user-guide/config.md#logging-configuration) for advanced configurations.
```

Comment:
Hi @kacole2,

I've previously noted for sake of readability and long-term support it's a documented request to limit Markdown to a width of 80 columns. Please reformat the above text. Thanks!